polyfactory/factories/base.py

```diff
@@ -184,10 +184,14 @@ class BaseFactory(ABC, Generic[T]):
     _factory_type_mapping: ClassVar[dict[Any, type[BaseFactory[Any]]]]
     _base_factories: ClassVar[list[type[BaseFactory[Any]]]]
 
+    _providers: ClassVar[dict[Any, Callable[[], Any]]]
+    "List of type providers that apply to all factories"
```

Comment:
```suggestion
    """Mapping of type providers that apply to all factories"""
```
update to match format of other class doc